templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -1238,11 +1269,6 @@ Resources:
         - IpProtocol: tcp
           FromPort: 22
           ToPort: 22
-          CidrIp:
```

Comment:
Was this removed on purpose?

Reply:
I have not rebased with the latest develop I think, I will do that in this push